apis/scheduling/config/types.go

```diff
@@ -123,3 +123,13 @@ type ReservationArgs struct {
 	// EnablePreemption indicates whether to enable preemption for reservations.
 	EnablePreemption *bool `json:"enablePreemption,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// GangSchedulingArgs defines the parameters for GangScheduling plugin.
+type GangSchedulingArgs struct {
```

Comment:
the plugin called `Gang` but the args called `GangSchedulingArgs`.   This implementation is invalid as required by the framework. It is necessary to keep the prefix of Args consistent with the plugin name.

Reply:
yes, just  call the prefix "Gang"